src/components/denhaag-wrappers/paginations/Paginations.tsx

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
The functionality of this Pagination component is much smaller than the one in the Den Haag storybook. Right now, I think we're missing:

- Link highlighting;
- Current page indicator;
- Disabled page indicators;
- Disabled prev / next buttons;
- ... 

For the full functionality you can check the storybook!